### ISSUE TITLE
tabSlideBox.js won't load. Getting "SyntaxError: expected expression, got '<' Line 4"

### ISSUE THREAD

**Issue by kehnee** [saravmajestic/ionic#16]
My paths are correct and I have the latest version of Ionic running. Any suggestions?

**Comment by saravmajestic** [comment]
Can you create a codepen? The samples http://ionic-sarav.rhcloud.com/ionic/tabbedSlideBox/default.html#/ are using RC4 and those are working.

**Comment by kehnee** [comment]
Actually I'm on Ionic v1.3.22. Sorry I just checked. Do you have a different version for Ionic v1.3 or it's the same?

**Comment by saravmajestic** [comment]
Ionic does not have v1.3 yet! Did you mean some other version number?

**Comment by kehnee** [comment]
_             _  
 (_)           (_)     

---

 | |/ _ | '_ | |/ **|
 | | (_) | | | | | (** 
 |_|___/|_| |_|_|___|  CLI v1.3.22

Usage: ionic task args

That's the ionic version from my command line. I recently upgraded and that's the version showing now. I must have first dibs. Lol

**Comment by saravmajestic** [comment]
Oh. Got it. You are mentioning ionic CLI version. I wasn't using ionic CLI. Is it loading nightly version for CSS and JS? Would you be able to create a codepen?

**Comment by kehnee** [comment]
I don't think it loads a nightly version of CSS and JS. Trying to create a codepen but might be hard to recreate the error with a codepen. Seems to be a conflict somewhere.
